testData/typeAlias/unionTypeParams.d.ts

```diff
@@ -0,0 +1,14 @@
+type Values<V> = List<V>;
+type ListOrMultiMap<K,V> = List<V> | Map<K,Values<V>>;
```

Comment:
Also, Kotlin's alias type isn't fully released yet.  I'm ok with using an early access for ts2kt, but not for the Kotlin it generates until that is released.